wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationRequestStrategy.swift

```diff
@@ -334,10 +334,32 @@ extension ConversationRequestStrategy: ZMUpstreamTranscoder {
         guard let newConversation = managedObject as? ZMConversation else {
             return false
         }
+        guard let apiVersion = APIVersion(rawValue: response.apiVersion) else { return false }
+        var knownError = false
+        defer {
+            if !knownError {
+                newConversation.notifyUnknownResponseError()
+            }
+        }
 
         if let responseFailure = Payload.ResponseFailure(response, decoder: .defaultDecoder),
            responseFailure.code == 412 && responseFailure.label == .missingLegalholdConsent {
             newConversation.notifyMissingLegalHoldConsent()
+            knownError = true
+        }
+
+        switch apiVersion {
+        case .v4:
+        if response.httpStatus == 409,
+           let nonFederatingFailure = Payload.NonFederatingBackends(response, decoder: .defaultDecoder),
+           nonFederatingFailure.nonFederatingBackends.count >= 2 {
```

Comment:
Why is the count important?

Reply:
There should be at least two backends if they do not federate with one another